src/Misc/layoutbin/en-US/strings.json

```diff
@@ -157,6 +157,8 @@
     "--auth                 Authentication type. Valid options are PAT (Personal Access Token),",
     "                       Negotiate (Kerberos or NTLM), Integrated (Windows default credentials) and",
     "                       ALT (Alternate Credentials)",
+    "--work                 Agent work folder, default to _work folder under agent root,",
```

Comment:
@ppanyukov - this is an excellent point.  We should definately mention that.

I would go farther and stamp the work dir as being owned by an agent.  Agent could gen guid id on initial config and we could stamp a file in the workdir.

For this checkin, adding the message is fine.